Alloy/commands/compile/ast/handle-alloy-globals.js

```diff
@@ -0,0 +1,98 @@
+const toCheck = [ 'Alloy', '_', 'Backbone' ];
+var template = require('@babel/template').default;
+
+var buildRequire = template(`
+	var VARIABLE = REQUIRECALL;
+`);
+module.exports = function(babel) {
+	return {
+		name: 'app.js top level variables global transform',
+		visitor: {
+			CallExpression: function (path, state) {
+				const node = path.node;
+				if (node.callee.name !== 'require') {
+					return;
+				}
+				if (!node.arguments || !node.arguments[0]) {
+					return;
+				}
+				switch (node.arguments[0].value) {
+					case '/alloy':
```

Comment:
Don't a lot of users have legacy/deprecated require calls with no leading '/'? May need to add extra cases without the leading '/'.